frontend/src/pages/Footer/_Footer.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./_Footer.scss";
```

Comment:
Per my comments on slack:

@Sagun Shrestha @Narek Tsaturyan I think I know how some of the styles are being overridden. The main reason is that index.scss is compiled last, so any styles in that takes precedence over everything else. So instead of doing this for importing:

Footer.js

```
import React, { Fragment } from "react";
import { Button } from "../Buttons/Buttons";
import "./_Footer.scss"; <----- DO NOT DO THIS!
```

Do this instead:

index.scss
```
@use "components/normalize";
@use "components/Basics" as *;
@use "components/Buttons/Buttons" as *;
@use "path/to/Footer.scss"; <------- DO THIS!
```

This is my fault as I have been doing things the first way, but with your current and future issues please do it the second way. I will separately move all imports over to index.scss and fix any bugs that come up and merge the fix after your issues are done.

I have also noticed that both you and @NickTSA does not seem to get announcements I make on slack about important code updates. Is there a prefered way I can communicate them to the both of you? I want to avoid making repeated announcements.

Reply:
I tried using this but the my styles seems to break
<img width="722" alt="Screen Shot 2022-02-28 at 4 48 56 PM" src="https://user-images.githubusercontent.com/77471354/156083768-4e134d69-8652-447f-be86-062c2008fee0.png">
![Screen Shot 2022-02-28 at 4 49 12 PM](https://user-images.githubusercontent.com/77471354/156083769-5e161b0e-822c-4269-bb2e-6a27af17a05d.png)
.

Reply:
@slavdefense I think you took my example too literally and ended up using a nonsense path url. Please look at documentation on on [`@use`](https://sass-lang.com/documentation/at-rules/use) for examples on properly configuring paths and the [last pr from main ](https://github.com/hackforla/CivicTechJobs/pull/166/files)to see what the changes are doing.